clusterloader2/pkg/measurement/common/network-policy/manifests/dep-test-client-policy-creation.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
```

Comment:
Maybe it would make sense to combine this file and `dep-test-client-pod-creation.yaml` as diff is only 3 lines and seems like all arguments are almost the same?

Reply:
Yes. I parameterized AppName and AppVersion, and merged the files into one.

Reply:
I reverted it to two files, because I actually couldn't make it work.
`command:` arguments couldn't be parsed correctly when some arguments are inside of `{{if}}`s.

Anyway I think it's good to have two files as these are two different applications, and it'll be easier to extend and maintain it this way.